cognite/client/data_classes/transformations/common.py

```diff
@@ -328,6 +447,49 @@ def dump(self, camel_case: bool = False) -> Dict[str, Any]:
         """
         return basic_obj_dump(self, camel_case)
 
+class SourceNonceCredentials:
+    def __init__(
+        self,
+        session_id: int,
+        nonce: str,
+        cdf_project_name: str,
+    ):
+        self.session_id = session_id
+        self.nonce = nonce
+        self.cdf_project_name = cdf_project_name
+
+    def dump(self, camel_case: bool = False) -> Dict[str, Any]:
+        """Dump the instance into a json serializable Python data type.
+
+        Args:
+            camel_case (bool): Use camelCase for attribute names. Defaults to False.
+
+        Returns:
+            Dict[str, Any]: A dictionary representation of the instance.
+        """
+        return basic_obj_dump(self, camel_case)
+
+class DestinationNonceCredentials:
+    def __init__(
+        self,
+        session_id: int,
+        nonce: str,
+        cdf_project_name: str,
+    ):
+        self.session_id = session_id
+        self.nonce = nonce
+        self.cdf_project_name = cdf_project_name
+
+    def dump(self, camel_case: bool = False) -> Dict[str, Any]:
+        """Dump the instance into a json serializable Python data type.
+
+        Args:
+            camel_case (bool): Use camelCase for attribute names. Defaults to False.
+
+        Returns:
+            Dict[str, Any]: A dictionary representation of the instance.
+        """
+        return basic_obj_dump(self, camel_case)
```

Comment:
These classes are exactly the same as `NonceCredentials`, so they're redundant. Please remove them and use `NonceCredentials` instead.